iceoryx_posh/source/runtime/posh_runtime.cpp

```diff
@@ -125,17 +125,17 @@ const RuntimeName_t& PoshRuntime::verifyInstanceName(cxx::optional<const Runtime
     if (!name.has_value())
     {
         LogError() << "Cannot initialize runtime. Application name has not been specified!";
-        std::terminate();
+        errorHandler(Error::kPOSH__RUNTIME_NO_NAME_PROVIDED, nullptr, ErrorLevel::FATAL);
```

Comment:
Please use every error enum only once and introduce some additional error enums here.

Reply:
It is still used twice, see line 133.